test/parallel/test-runner-run.mjs

```diff
@@ -68,10 +69,10 @@ describe('require(\'node:test\').run', { concurrency: true }, () => {
     const result = await run({
       files: [join(testFixtures, 'default-behavior/test/random.cjs')]
     }).compose(dot).toArray();
-    assert.deepStrictEqual(result, [
-      '.',
-      '\n',
-    ]);
+
+    assert.strictEqual(result.length, 2);
+    assert.strictEqual(util.stripVTControlCharacters(result[0]), '.');
+    assert.strictEqual(result[1], '\n');
```

Comment:
```suggestion
    assert.deepStrictEqual(result.map((v, i) => i === 0 ? util.stripVTControlCharacters(v) : v), [
      '.',
      '\n',
    ]);
```

Reply:
@aduh95 I am 👎 on this. For 2 lines a loop is just overkill and introduces confusion IMHO